etc/profile-a-l/librewolf.profile

```diff
@@ -37,7 +37,7 @@ include whitelist-usr-share-common.inc
 #private-etc librewolf
 
 dbus-user filter
-dbus-user.own org.mozilla.librewolf.*
+dbus-user.own io.gitlab.librewolf.*
```

Comment:
I'd keep both entries so that the profile works with both old and new versions
of LibreWolf.